package.json

```diff
@@ -114,7 +112,8 @@
     "tmp": "0.0.31",
     "umount": "^1.1.6",
     "unzip2": "^0.2.5",
-    "update-notifier": "^2.2.0"
+    "update-notifier": "^2.2.0",
+    "analytics": "github:resin-io-playground/node.analytics"
```

Comment:
Hmmm, does it "make sense" for a project in the `resin-io` org to have a dependency on a project in the `resin-io-playground` org? I dunno if there's any kind of policy around this ( @lekkas ?) but if `resin-cli` is depending on `node.analytics`, then perhaps it makes sense to move `node.analytics` from `resin-io-playground` to `resin-io-modules` ?

Reply:
we will move node.analytics to another repo down the line. I want to make sure everything works as expected. I then support I'll move the analytics lib to another repo, publish to npm and then merge this PR.

Reply:
Yep, this is fine for review, but I'd definitely like to fix this and release node.analytics properly before we merge this PR.